js/chart.js

```diff
@@ -1,22 +1,87 @@
-function Chart(){
-    var renderChart = function(){
-        //code to render charts
-    }
-    this.coloumnChart = function(){
+function Chart() {
+
+    this.coloumnChart = function () {
         return 'yet not decided';
```

Comment:
this.render(chartType)  =function() {

// decide which chart object to be created and to render
var x = new XChart();
x.renderChart();
}